TypeCobol.LanguageServer/WorkspaceProject.cs

```diff
@@ -0,0 +1,202 @@
+        private HashSet<string> GetCopyFolders()
+        {
+            var result = new HashSet<string>(StringComparer.InvariantCultureIgnoreCase);
+            foreach (var cobolLibrary in Project.SourceFileProvider.CobolLibraries)
+            {
+                if (cobolLibrary is LocalDirectoryLibrary localDirectoryLibrary)
+                {
+                    result.Add(localDirectoryLibrary.RootDirectory.FullName);
+                }
+            }
+
+            return result;
+        }
+
+        /// <summary>
+        /// Compare two Copy folders set
+        /// </summary>
+        /// <param name="oldCopyFolders">The Old set</param>
+        /// <param name="newCopyFolders">The nes set</param>
+        /// <returns></returns>
+        private static bool AreCopyFoldersSame(HashSet<string> oldCopyFolders, List<string> newCopyFolders)
+        {
+            if (oldCopyFolders.Count != newCopyFolders.Count)
+                return true;// The size of the list are different
+
+            foreach (var newCopyFolder in newCopyFolders)
+            {
+                string fullPath = Path.GetFullPath(newCopyFolder);
+                if (!oldCopyFolders.Contains(fullPath))
+                    return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Configure this project
+        /// </summary>
+        /// <param name="format">The new Document format if any, null otherwise</param>
+        /// <param name="options">The new options if any, null otherwise</param>
+        /// <param name="analyzerProviderWrapper">The new Analyzer provider if any, null otherwise</param>
+        /// <param name="copyFolders">The new list of copy folders associated to this project if any, null otherwise</param>
+        public void Configure(DocumentFormat format, TypeCobolOptions options, AnalyzerProviderWrapper analyzerProviderWrapper, List<string> copyFolders)
+        {
+            bool updateFormat = false;
+            var newFormat = Project.Format;
+            if (format != null)
+            {
+                updateFormat = true;
+                newFormat = format;
+            }
+
+            bool updateOptions = false;
+            var newOptions = Project.CompilationOptions;
+            if (options != null)
+            {
+                updateOptions = true;
+                newOptions = options;
+            }
+
+            bool updateAnalyzerProvider = false;
+            var newAnalyzerProvider = Project.AnalyzerProvider;
+            if (analyzerProviderWrapper != null)
+            {
+                updateAnalyzerProvider = true;
+                newAnalyzerProvider = analyzerProviderWrapper;
+            }
+
+            var oldCopyFolders = GetCopyFolders();
+            bool updateCopyFolders = false;
+            var newCopyFolders = oldCopyFolders.ToList();
+            if (copyFolders != null && AreCopyFoldersSame(oldCopyFolders, copyFolders))
```

Comment:
Missing '!' before `AreCopyFoldersSame(oldCopyFolders, copyFolders)`

Reply:
Changed in [8d9c2da](https://github.com/TypeCobolTeam/TypeCobol/pull/2158/commits/8d9c2daba074a18dbf5260d351d3278d470ebf44)